src/main/java/pl/pop/interview/master/question/QuestionFacade.java

```diff
@@ -23,6 +23,12 @@ public List<QuestionDTO> getAllQuestions() {
                 .toList();
     }
 
+    public List<QuestionDTO> getQuestionsAnsweredByPractitioner(Long practitionerId) {
```

Comment:
tak na dobrą sprawę, to tutaj powinno się zwracać Page< QuestionDTO >

Reply:
OK - teraz metoda getQuestionsForPractitioner zwraca Page<QuestionDTO>